### PR TITLE
react hw, basic react & props

### PR DESCRIPTION
**Pull request by m-elshawa**
React Week1 hw (without Fibonacci exercise)

### CODE REVIEW

File: react/week1/todo-list-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import Getup from './Components/Getup'
+import Brush from './Components/Brush';
+import Eat from './Components/Eat';
+import Useclasses from './Components/Useclasses'
+
+function App() {
+  return (
+    <div className="App">
+      <Todolist />
+      {/* <Getup description = "Get out of bed" deadline = "Wed Sep 13 2017" /> */}
```
Comment (piyushpalod) [inline react/week1/todo-list-app/src/App.js:13]: we need to create components which will take data from parent to child and show data, here you are calling the same component multiple times, can you make it in loop and provide array of data. Remember whats Niels said in class its all about separation

File: react/week1/todo-list-app/src/Components/Brush.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Brush = props => {
+    console.log(props)
+    return (
+        <div>
+           <li>{props.description} {":"} {props.deadline}</li>
```
Comment (piyushpalod) [inline react/week1/todo-list-app/src/Components/Brush.js:7]: You dont need to create component of each string item, instant create component which take array and show it. so consider creating component like Todo >  TodoList > TodoItem

**Review by piyushpalod — CHANGES_REQUESTED** [review]
Please consider to make component on the basis of showing items, Not for each string.

File: react/week1/todo-list-app/src/App.js (hunk 1 above)
Comment (m-elshawa) [reply react/week1/todo-list-app/src/App.js:13]: Thanks for your feedback Piyush, but unfortunately I'm lost, I tried to use the map method to do it in a loop but I failed. so I donno how to this :(

**Pull request merged**